pkgs/development/python-modules/jupyterlab-git/default.nix

```diff
@@ -72,5 +78,7 @@ buildPythonPackage rec {
     homepage = "https://github.com/jupyterlab/jupyterlab-git";
     license = with licenses; [ bsd3 ];
     maintainers = with maintainers; [ chiroptical ];
+    # https://github.com/jupyterlab/jupyterlab-git/issues/1245
```

Comment:
[0.50.0a2](https://github.com/jupyterlab/jupyterlab-git/releases/tag/v0.50.0a2) supports lupyerlab-4

Reply:
I will wait for the official release 0.50.0, but may adopt it in the ZHF.